terraform/nifi.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_iam_user" "nifi" {
+  name = "${local.prefix}-nifi"
```

Comment:
Prefix var will give us a name for the user as: <app>-<stage>-nifi Do we want to include in the beginning a workspace also to keep it consistent?